src/irmin-pack/unix/file_manager.ml

```diff
@@ -185,7 +185,7 @@ struct
   (** Is expected to be called by the index when its append buffer is full so
       that the dependendies of index are flushes. When the function returns,
       index will flush itself. *)
-  let index_is_about_to_auto_flush_exn t =
+  let index_is_about_to_flush_exn t =
```

Comment:
I liked the distinction between `auto flush` and `flush` (which should have maybe been called `explicit flush`). 

In the case of the above function it's important to keep the distinction. It will only be called during auto flushes and not during explicit flushes